src/dialects/sqlite/query-generator.js

```diff
@@ -366,18 +366,10 @@ export class SqliteQueryGenerator extends SqliteQueryGeneratorTypeScript {
     return result;
   }
 
-  showConstraintsQuery(tableName, constraintName) {
-    let sql = `SELECT sql FROM sqlite_master WHERE tbl_name='${tableName}'`;
-
-    if (constraintName) {
-      sql += ` AND sql LIKE '%${constraintName}%'`;
-    }
-
-    return `${sql};`;
-  }
-
   describeCreateTableQuery(tableName) {
```

Comment:
Can you add the typings for this as well?

Reply:
Done